From 1236dde83a16897ed301466b7ee3ae1cc7c19f94 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Fri, 1 Jun 2012 17:07:36 +0800
Subject: [PATCH 284/502] ENGR00212226 add clear screen function for primary
 display.

add three display support for settings dispd and display manager service.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 mx6/libgralloc_wrapper/framebuffer.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mx6/libgralloc_wrapper/framebuffer.cpp b/mx6/libgralloc_wrapper/framebuffer.cpp
index f4c044a..e2a82b6 100755
--- a/mx6/libgralloc_wrapper/framebuffer.cpp
+++ b/mx6/libgralloc_wrapper/framebuffer.cpp
@@ -736,6 +736,8 @@ int fb_device_open(hw_module_t const* module, const char* name,
             dev->priv_m = m;
             dev->mainDisp_fd = m->framebuffer->fd;
             dev->isMainDisp = 1;
+            gralloc_module_t* gr_m = reinterpret_cast<gralloc_module_t*>(m);
+            gr_m->perform = fb_perform;
             fslwatermark_sem_open();
         } else {
             private_module_t* orig_m = (private_module_t*)module;
-- 
1.8.0

